src/components/profile-page/settings/profile-display-name.tsx

```diff
@@ -5,19 +5,23 @@
  */
 
 import type { ChangeEvent, FormEvent } from 'react'
-import React, { useEffect, useState } from 'react'
-import { Alert, Button, Card, Form } from 'react-bootstrap'
+import React, { useCallback, useEffect, useState } from 'react'
+import { Button, Card, Form } from 'react-bootstrap'
 import { Trans, useTranslation } from 'react-i18next'
 import { updateDisplayName } from '../../../api/me'
 import { fetchAndSetUser } from '../../login-page/auth/utils'
 import { useApplicationState } from '../../../hooks/common/use-application-state'
+import { showErrorNotification } from '../../../redux/ui-notifications/methods'
 
+const REGEX_INVALID_DISPLAY_NAME = /^\s*$/
```

Comment:
maybe it would be desireable to use `REGEX_VALID_DISPLAY_NAME` here, too? For consistencies sake?

Reply:
As this regex does nothing more than ensuring that the display name is not empty, I replaced it with a check `.trim() !== ''`.